geopandas/io/file.py

```diff
@@ -113,16 +113,19 @@ def infer_schema(df):
     except ImportError:
         from ordereddict import OrderedDict
 
-    def convert_type(in_type):
+    def convert_type(column, in_type):
         if in_type == object:
             return 'str'
         out_type = type(np.asscalar(np.zeros(1, in_type))).__name__
         if out_type == 'long':
             out_type = 'int'
+        if out_type == 'bool':
+            raise ValueError("column {} is boolean type, ".format(column) +
```

Comment:
maybe put quotes (`'{}'`) around the column name it make it clearer it is referring to that ?